internal/appsec/waf_test.go

```diff
@@ -24,6 +24,45 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func TestCustomRules(t *testing.T) {
+	t.Setenv("DD_APPSEC_RULES", "testdata/custom_rules.json")
+	appsec.Start()
+	defer appsec.Stop()
+
+	if !appsec.Enabled() {
+		t.Skip("appsec disabled")
+	}
+
+	// Start and trace an HTTP server
+	mux := httptrace.NewServeMux()
+	mux.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
+		w.Write([]byte("Hello World!\n"))
+	})
+
+	srv := httptest.NewServer(mux)
+	defer srv.Close()
+
+	t.Run("method", func(t *testing.T) {
+		mt := mocktracer.Start()
+		defer mt.Stop()
+
+		req, err := http.NewRequest("POST", srv.URL, nil)
+		if err != nil {
```

Comment:
Let's go with `require.NoError(...)` for consistency

Reply:
done 👍